api/udpa/service/orca/v1/orca.proto

```diff
@@ -0,0 +1,35 @@
+syntax = "proto3";
+
+package udpa.service.orca.v1;
+
+option java_outer_classname = "OrcaProto";
+option java_multiple_files = true;
+option java_package = "io.envoyproxy.udpa.service.orca.v1";
+option go_package = "v1";
+
+import "udpa/data/orca/v1/orca_load_report.proto";
+
+import "google/protobuf/duration.proto";
+
+import "validate/validate.proto";
+
+// Out-of-band (OOB) load reporting service for the additional load reporting
```

Comment:
Can you add a pointer and reference to the public design doc and relevant section?

Reply:
ACK.